### DISCUSSION TITLE
Can not to override Empty data warning message depending on search result.

### DISCUSSION THREAD

**Discussion by alizzada** [ghiscoding/Angular-Slickgrid#939] [Q&A] [answered]
Hi, in my case I enabled empty data warning message and overrided like this.

`enableEmptyDataWarningMessage: true,
            emptyDataWarning: {
                message: 'No Records Found'
            }`

But I want to also override this message depending on my search result. For example if returns >10k data then this message should be overridden like this. 'Your search returns too many results. Please refine your filter options.' For this case I'm using this code

` this.gridOptions.emptyDataWarning.message = 'Your search returns too many results. Please refine your filter options.';`

Unfortunately  message doesn't changes and remaining 'No Records Found'. 
Actually problem is that  

` this.gridOptions.emptyDataWarning.message`   couldn't override initial message. 

Is there any solutions?

**Comment by ghiscoding** [comment, marked as answer]
it wasn't built to be changed dynamically, it was created to only keep 1 static text, the only dynamic portion is the locale translation

**Reply by alizzada** [reply]
Thanks for reply.

**Comment by alizzada** [comment]
I tried to get good result with some logics and eventually I did it like as below.

` const emptyDataWarningMessages = document.getElementsByClassName('slick-empty-data-warning left');`

Depending on my if statement I wrote like as

`if ( your condition ) {
 for (let i = 0; i < emptyDataWarningMessages.length; i++) {
                    const emptyDataWarningMessage = emptyDataWarningMessages[i] as HTMLElement;
                    emptyDataWarningMessage.textContent = 'Your search returns too many results. Please refine your filter options.'
                }
}`
